code/game/machinery/cryopod.dm

```diff
@@ -372,9 +372,10 @@ GLOBAL_LIST_INIT(frozen_items, list(SQUAD_MARINE_1 = list(), SQUAD_MARINE_2 = li
 		occupant.ghostize(0)
 
 	//Make an announcement and log the person entering storage.
-	GLOB.frozen_crew += "[occupant.real_name]"
+	GLOB.frozen_crew += "[occupant.real_name] ([occupant.comm_title])"
```

Comment:
```suggestion
	GLOB.frozen_crew += "[occupant.real_name] ([occupant.job])"
```
It is probably fine either way, but using full name of the job instead of shorthand feels more appropriate...